app/common/EnrolmentCommon.scala

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
```

Comment:
there's a warning about the header not matching the expected text. I think it might need the same copyright header that's in the other repos.

Reply:
updated for all